ajax/libs/rrssb/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "rrssb",
+  "filename": "rrssb.min.js",
+  "version": "1.14.0",
+  "author": "Daniel Box <daniel@kurtnoble.com> (http://github.com/kni-labs)",
+  "description": "Ridiculously Responsive Social Sharing Buttons",
+  "keywords": [
+    "ridiculously",
+    "RRSSB",
+    "rrssb",
+    "responsive",
+    "sharing",
+    "social",
+    "buttons"
```

Comment:
Would you please add more keywords?
![2018-02-19 10 30 58](https://user-images.githubusercontent.com/8349587/36360550-0bd8cd58-1560-11e8-8f15-65569e49dcdf.png)
Thanks